app/services/course/assessment/answer/programming_auto_grading_service.rb

```diff
@@ -42,6 +42,7 @@ def build_submission_files(answer)
   # @param [Course::Assessment::ProgrammingPackage] package The package to import.
   # @return [Course::Assessment::ProgrammingEvaluationService::Result]
   def evaluate_package(question, package)
+    package.save
```

Comment:
Okay, but I think the logic should be handled outside, it's out of the duty of evaluate package, what do you think ?